src/models/Card/Card.ts

```diff
@@ -1,15 +1,27 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-unused-vars */
+export enum Group {
+  MaterialGroup = "MATERIALSUPPLIERS",
+  ConvenienceStores = "CONVENIENCESTORES",
+  FoodAndBeverage = "FOODANDBEVERAGE",
+  Equipment = "EQUIPMENT",
+  OfficeSupplies ="OFFICESUPPLIESANDCLEANING",
+  FuelAndGas = "FUELANDGAS",
+}
```

Comment:
no need to change this yet but we probably need to get this from the api instead of hard coding it